src/GitHub.VisualStudio/TeamExplorer/Connect/GitHubConnectSection.cs

```diff
@@ -377,7 +378,7 @@ public SectionStateTracker(ITeamExplorerSection section, Action onRefreshed)
                     .Ignore("Title")
                     .PermitIf("IsBusy", SectionState.Idle, () => !this.section.IsBusy)
                     .IgnoreIf("IsBusy", () => this.section.IsBusy)
-                    .OnExit(onRefreshed);
+                    .OnExit(() => onRefreshed());
```

Comment:
Nope, the compiler doesn't recognize this as being an potentially awaited usage so it doesn't complain.